tools/scripts/pttestscript

```diff
@@ -3,4 +3,4 @@
 print("hello")
 
 if __name__ == '__main__':
-    print("This is the test file")
+    print("This is the test file" ) 
```

Comment:
If files don't end with `.py` they are not formatted...

Reply:
Is there a way to change that? Why does this file not end with `.py?`

Reply:
There is a regexp. I made it without py as it slightly nicer looking (IMO) to run programs in the terminal without extensions

Reply:
Maybe we can change this, or just ignore there few files that are made to run as scripts. Anyone with vscode will format those files anyway. It is just the hook.

Reply:
I was thinking we could add exceptions to these files as well. So that we include all files that end with .py + (pttestscript|...|...). Is that possible?

Reply:
I added a better regexp now 👍